Dockerfile.search.public

```diff
@@ -8,3 +8,21 @@ COPY requirements-common.txt /app/requirements-common.txt
 RUN pip3 install -e .
 
 CMD [ "python3", "search_service/search_wsgi.py" ]
+
+FROM base as oidc-release
+
+RUN  pip3 install -e .&& \
+     pip3 install -e .[oidc]
+
+ENV FLASK_APP_MODULE_NAME flaskoidc
+ENV FLASK_APP_CLASS_NAME FlaskOIDC
+ENV FLASK_OIDC_WHITELISTED_ENDPOINTS status,healthcheck,health
+ENV SQLALCHEMY_DATABASE_URI sqlite:///sessions.db
+
+# You will need to set these environment variables in order to use the oidc image
```

Comment:
You also need to add `FLASK_OIDC_CONFIG_URL`
Would be nice to add this link for ref: https://github.com/verdan/flaskoidc#readme

This change should go across all repos.

Reply:
@verdan let me know if this is resolved